src/components/application-loader/initializers/configLoader.ts

```diff
@@ -21,14 +23,8 @@ export const loadAllConfig: (baseUrl: string) => Promise<void> = async (baseUrl)
   }
   setConfig(config)
 
-  const banner = config.banner
-  if (banner.text !== '') {
-    const lastAcknowledgedTimestamp = window.localStorage.getItem('bannerTimeStamp') || ''
-    setBanner({
-      ...banner,
-      show: banner.text !== '' && banner.timestamp !== lastAcknowledgedTimestamp
-    })
-  }
+  fetchAndSetBanner(backendUrl)
+    .catch(console.error.bind(this))
```

Comment:
As this does not belong to the config anymore, why do you include it into the configLoader.ts instead of adding it as an own initializer task?

Reply:
There was a reason why I did that.. but i don't know why anymore. I'll fix that.